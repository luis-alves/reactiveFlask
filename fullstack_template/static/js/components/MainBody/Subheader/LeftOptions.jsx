import React from "react"

export default class LeftOptions extends React.Component {
  render () {
    return (
      <div className="left-options-main">
        <button><i className="icons icon-add"></i><i className="subheader-text">Add a transaction</i></button>
        <button><i className="icons icon-pencil"></i><i className="subheader-text">Edit</i></button>
        <button><i className="icons icon-download"></i><i className="subheader-text">Import</i></button>
        <button><i className="icons icon-undo"></i><i className="subheader-text">Undo</i>.</button>
        <button><i className="icons icon-redo"></i><i className="subheader-text">Redo</i></button>
      </div>
    )
  }
}
