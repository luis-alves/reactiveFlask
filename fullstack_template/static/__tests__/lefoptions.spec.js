import React from 'react';
import expect from 'expect';
import { shallow } from 'enzyme';
import LeftOptions from '../js/components/main/body/headers/subheader/LeftOptions.jsx';

describe('LeftOptions component', () => {
  it('renders without exploding', () => {
    expect(
      shallow(<LeftOptions />).length).toEqual(1)
  })
  // it('Header has class page-header', () => {
  //   const component = shallow(<Header />);
  //   expect(component.find('.page-header').length).toEqual(1);
  // })
})

test('Check snapshot', () => {
  const wrapper = shallow(<LeftOptions />);
  expect(wrapper).toMatchSnapshot();
});
