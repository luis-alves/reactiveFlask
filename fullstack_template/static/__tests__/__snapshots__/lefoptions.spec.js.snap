// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check snapshot 1`] = `
<div
  className="left-options-main"
>
  <button>
    <i
      className="icons icon-add"
    />
    <i
      className="subheader-text"
    >
      Add a transaction
    </i>
  </button>
  <button>
    <i
      className="icons icon-pencil"
    />
    <i
      className="subheader-text"
    >
      Edit
    </i>
  </button>
  <button>
    <i
      className="icons icon-download"
    />
    <i
      className="subheader-text"
    >
      Import
    </i>
  </button>
  <button>
    <i
      className="icons icon-undo"
    />
    <i
      className="subheader-text"
    >
      Undo
    </i>
    .
  </button>
  <button>
    <i
      className="icons icon-redo"
    />
    <i
      className="subheader-text"
    >
      Redo
    </i>
  </button>
</div>
`;
